react/features/filmstrip/components/web/Filmstrip.tsx

```diff
@@ -446,6 +447,12 @@ class Filmstrip extends PureComponent <IProps, State> {
                     _verticalViewGrid && 'no-vertical-padding',
                     _verticalViewBackground && classes.filmstripBackground) }
                 style = { filmstripStyle }>
+                <span
```

Comment:
Why is this useful here? I am using VoiceOver and it does not read/see this section.

Reply:
That's weird, works great for me… Don't you see "Video thumbnails" in the headings list if you open the VoiceOver rotor (via the VO+U shortcut)?

Reply:
Will check it.

Reply:
All good!
I was more familiar with NVDA than with VoiceOver. 
Thank you for the indications!